### PR TITLE
new jovo communication rules

### PR DESCRIPTION
**Pull request by sampan501**

### CODE REVIEW

**Review by PSSF23 — DISMISSED** [review]
👍

**Review by jdey4 — DISMISSED** [review]

File: content/about/agreements.md [hunk 1]
```diff
@@ -241,7 +241,14 @@ Three months prior to departure, make a 'transition' plan, wherein the technical
 
 #### Authorship Agreements
 
-Typically, the person actually submitting the paper is the first author.  This means that sometimes, if we get feedback after an individual has left the lab, and that person decides to prioritize other work, the 2nd author may be responsible for resubmitting, and the author order may switch accordingly.  
+Typically, the person actually submitting the paper is the first author.  This means that sometimes, if we get feedback after an individual has left the lab, and that person decides to prioritize other work, the 2nd author may be responsible for resubmitting, and the author order may switch accordingly.
+
+#### Communication
+
+1. The primary ways for a NeuroData family member to contact jovo will be scheduling 1-on-1 meetings on [calendly](https://calendly.com/joshuav), or direct messages on Slack and text
```
Comment (bdpedigo) [inline content/about/agreements.md]: can we also schedule via just putting on the neurodata calendar?

Comment (sampan501) [reply content/about/agreements.md]: yea I can edit that

Comment (jovo) [reply content/about/agreements.md]: yes.

**Review by jovo — CHANGES_REQUESTED** [review]
clarify point from pedigo and merge

**Review by bdpedigo — APPROVED** [review]

**Pull request merged**